tinker/templates/events/form.html

```diff
@@ -339,19 +339,29 @@ <h3 class="subtitle">{{ field.label }}</h3>
             };
 
 
-            function offCampusLocation() {
+            function offCampusLocation(clear_values) {
```

Comment:
Can you set a default value to clear_values? It should just be something like:

`function offCampusLocation(clear_values=false) {`

Reply:
We tried this out a couple of different ways and decided it was more trouble than it was worth.